upgrade.sh

```diff
@@ -12,7 +12,7 @@ KUBECONFIG=aks/admin.kubeconfig helm upgrade $RESOURCEGROUP pkg/helm/chart -f _d
 
 # TODO: when sync runs as an HCP pod (i.e. not in development), hopefully should
 # be able to use helm upgrade --wait here
-for d in master-etcd master-api master-controllers; do
+for d in master-api master-controllers; do
```

Comment:
OK, it seems we need to think about etcd upgrades separately. Follow-up issue for adding an etcd-upgrade.sh script? I expect in prod we are going to separate those anyway.

Reply:
We can helm resource to manage CRD. We just need different checking method to validate if it was updated ok.